### ISSUE TITLE
Unhandled error when `gg_playback()` is run after no plots produced

### ISSUE THREAD

**Issue by TylerGrantSmith** [thebioengineer/camcorder#11]
``` r
dir <- fs::file_temp("camcorder")
camcorder::gg_record(dir = dir)
camcorder::gg_playback()
#> Error in normalizePath(path.expand(path), winslash, mustWork): path[1]=NA
```

**Comment by thebioengineer** [comment]
Thanks for pointing that out. I hadn't considered someone starting/stopping without making any plots!

**Comment by thebioengineer** [comment]
fixed in PR #13
